src/cli/config-flags.ts

```diff
@@ -35,12 +35,65 @@ export const BOOLEAN_CLI_ARGS = [
   'verbose',
   'version',
   'watch',
+
+  // JEST CLI OPTIONS
+  'all',
+  'automock',
+  'bail',
+  // 'cache', Stencil already supports this argument
```

Comment:
some of these options are dupes of options that Stencil already supports. I thought we might want to keep them in the list here just so that the list is, in the code at least, a complete list of the Jest CLI arguments. Open to just deleting these commented-out lines as well though!